source/administration/data-retention.rst

```diff
@@ -108,3 +108,19 @@ Does the data retention job include archived channels?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Posts and attachments in archived channels are affected by the data retention job. If a post exceeds the age configured for the data retention job it will be deleted from the database.
+
+How long does it take to run a deletion query and does it affect server performance?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Data retention runs the actual deletion query in batches, deleting data in blocks of 1,000 records per query so the database won’t be locked up for extended periods of time with slow, long-running queries (keeping to this limit keeps the query down to a few milliseconds execution time on the database itself).
+
+Each batch of data is deleted based on indexes, so the queries are generally quite quick to execute on such small batches meaning the server will stay fully responsive while the process is running.
```

Comment:
```suggestion
Each batch of data is deleted based on indexes, meaning the queries are generally quick to execute on such small batches. This helps the server remain fully responsive while the process is running.
```

Reply:
Each batch of data is deleted based on indexes making the queries quick to execute on small batches. This helps the server remain fully responsive while the process is running

@justinegeffen here is my suggestion what do you think?

Reply:
I really like it, thanks. I've added a new suggestion with your change. :)